src/patternfly/components/Alert/alert.scss

```diff
@@ -83,14 +91,26 @@
   background-color: var(--pf-c-alert--BackgroundColor);
   box-shadow: var(--pf-c-alert--BoxShadow);
 
+  // left border treatment base
+  // color is transparent by default, then set by the state modifier classes and turned on by the inline modifier
+  &::before {
+    position: absolute;
+    top: var(--pf-c-alert--before--Top);
+    left: 0;
+    z-index: var(--pf-c-alert--before--ZIndex);
```

Comment:
Do you need the z-index? Doesn't look like it's necessary. Plus with this high of a z-index, I believe if another component with a lower z-index (like a dropdown menu with `--pf-global--ZIndex--xs`) pops up over the alert, this border would show up on top of that component.